genny/build/assets_test.go

```diff
@@ -37,6 +38,11 @@ func Test_assets(t *testing.T) {
 	for i, c := range res.Commands {
 		r.Equal(cmds[i], strings.Join(c.Args, " "))
 	}
+
+	s, err := res.Find("../public/assets/dummy")
+	if err != nil || s.String() != "placeholder for static builds" {
+		panic(fmt.Sprintf("%v", s.String()))
```

Comment:
Please follow the rest of the tests and follow the pattern accordingly. We never use `panic`. Thanks.

Reply:
:) yup... forgot I just did this for debugging . I’ll replace with a normal failure .